server/settings/simple_creators.py

```diff
@@ -1,13 +1,16 @@
+from pydantic import root_validator
+
 from ayon_server.settings import BaseSettingsModel, SettingsField
 
 
 class SimpleCreatorPlugin(BaseSettingsModel):
-    _layout = "expanded"
-    product_type: str = SettingsField("", title="Product type")
+
+    name: str = SettingsField(title="Name", disabled=True)
+    label: str = SettingsField("", title="Label")
     # TODO add placeholder
     identifier: str = SettingsField("", title="Identifier")
-    label: str = SettingsField("", title="Label")
     icon: str = SettingsField("", title="Icon")
+    product_type: str = SettingsField("", title="Product type")
```

Comment:
Why did you move the most important thing at the end of the item?

Reply:
Because this is more familiar to me. 
when implementing a creator, we first set the label, identifier icon and then add the product type.

Reply:
although the product type is more important to AYON. label can be more important for users. because they will see it in the publisher UI.
![image](https://github.com/user-attachments/assets/3a78aeef-edaf-4700-8c7a-f5de6cd1070d)

Reply:
The only required field is product type and extensions, the others are optional...

Reply:
let me try one more time.

Reply:
Reverted the change in order.
https://github.com/ynput/ayon-traypublisher/pull/44/commits/03df7b381548c5ca73b89700da93f3e833b6e9a1